Makefile

```diff
@@ -291,7 +291,7 @@ kind-reset: ## Destroys the "ipam" kind cluster.
 
 RELEASE_TAG ?= $(shell git describe --abbrev=0 2>/dev/null)
 RELEASE_DIR := out
-RELEASE_NOTES_DIR := releasenotes
+RELEASE_NOTES_DIR := CHANGELOG
```

Comment:
Does this dir actually matter what its called?

Reply:
No, it does not matter, we have to just make sure that this folder name is the same as the one we are checking in the workflow script. If we change it here we have to change in workflow script too. 
In my opinion CHANGELOG is ok, but we can change it if you have some other suggestions.

Reply:
Well, we have releasenotes dir already, which currently is just a placeholder. IMO it is just as valid name as changelog would be, without requiring any change here.

Reply:
changed it back to `releasenotes`, also removed `releasenotes` from gitignore.